docs/checkpoint.md

```diff
@@ -40,4 +40,4 @@ A local shared buffer is introduced to stage these uncommitted write batches. On
 
 ![shared buffer](./images/checkpoint/shared-buffer.svg)
 
-Another benefit of shared buffer is that the write batches in a compute node can be compacted into a single SSTable file before uploading, which significantly reduces the number of SSTable files in Layer 0.
```

Comment:
We may keep the name unchanged in docs?

Reply:
Why don't we unify it in code and documentation?

Reply:
I'd prefer just calling it `SST` in docs 🤣

Reply:
Since `SSTable` is more common in other docs and papers. Another example: we may use `Http` in Rust code while should keep `HTTP` in docs by convention.

Reply:
ok I will revert changes to the document